LocalPackages/NetworkProtectionMac/Sources/NetworkProtectionUI/NetworkProtectionPopover.swift

```diff
@@ -20,27 +20,10 @@ import AppKit
 import Combine
 import Foundation
 import SwiftUI
+import SwiftUIExtensions
 import NetworkProtection
 import LoginItems
 
-@available(iOS, introduced: 10.15, deprecated: 12.0, message: "Use Apple's DismissAction")
-public struct DismissAction: EnvironmentKey {
-    public static var defaultValue: () -> Void = {}
-}
-
-@available(iOS, introduced: 10.15, deprecated: 12.0, message: "Use Apple's DismissAction")
-public extension EnvironmentValues {
-    var dismiss: () -> Void {
-        get {
-            self[DismissAction.self]
-        }
-
-        set {
-            self[DismissAction.self] = newValue
-        }
-    }
-}
-
```

Comment:
This was previously only used for the NetP UI package, but it's generic enough that it makes sense to move to the SwiftUI extensions package.